src/ray/gcs/redis_actor_state_accessor.h

```diff
@@ -0,0 +1,101 @@
+#ifndef RAY_GCS_REDIS_ACTOR_STATE_ACCESSOR_H
+#define RAY_GCS_REDIS_ACTOR_STATE_ACCESSOR_H
+
+#include "ray/common/id.h"
+#include "ray/gcs/actor_state_accessor.h"
+#include "ray/gcs/callback.h"
+#include "ray/gcs/subscription_executor.h"
+#include "ray/gcs/tables.h"
+
+namespace ray {
+
+namespace gcs {
+
+class RedisGcsClient;
+
+/// \class RedisActorStateAccessor
+/// RedisActorStateAccessor class encapsulates the implementation details of
+/// reading or writing or subscribing of actor's specification (immutable fields which
+/// determined at submission time, and mutable fields which are determined at runtime).
+class RedisActorStateAccessor : public ActorStateAccessor {
+ public:
+  explicit RedisActorStateAccessor(RedisGcsClient *client_impl);
+
+  virtual ~RedisActorStateAccessor() {}
+
+  /// Get actor specification from GCS asynchronously.
+  ///
+  /// \param actor_id The ID of actor to look up in the GCS.
+  /// \param callback Callback that will be called after lookup finishes.
+  /// \return Status
```

Comment:
For documenting the method of a sub-class, if it doesn't have any differences with the super class, I prefer just say `see super class`. In case we change one place, and forget to change the other one.

Reply:
Good point.

Reply:
done